### PR TITLE
Continous CQL loss logging and aligning with discrete logging

### PR DESCRIPTION
**Pull request by joshuaspear**
Implementing logging for the conservative loss part of CQL for continuous CQL. Also altered the logging of the loss in the discrete model to include the value of $\alpha$

### CODE REVIEW

File: d3rlpy/algos/qlearning/torch/cql_impl.py [hunk 1]
```diff
@@ -75,7 +75,23 @@ def compute_critic_loss(
         conservative_loss = self._compute_conservative_loss(
             batch.observations, batch.actions, batch.next_observations
         )
-        return loss + conservative_loss
+        return loss + conservative_loss, conservative_loss
+
+    @train_api
+    def update_critic(self, batch: TorchMiniBatch) -> np.array:
```
Comment (takuseno) [inline d3rlpy/algos/qlearning/torch/cql_impl.py:81]: This is altering a signature of `update_critic`, which is `def update_critic(self, batch: TorchMiniBatch) -> float:`. Thus, we actually need to refactor these methods first. I'll take the first pass to return `Dict[str, float]`. Then, you can make changes on top of it. Sorry for the inconvenience. I'll let you know when it's done.

**Review by takuseno — COMMENTED** [review]
Thank you for your PR! As I commented below, I'll take the first pass to refactor signatures of methods. Then, I'll let you known when it's done.

File: .gitignore [hunk 2]
```diff
@@ -14,7 +14,7 @@ docs/d3rlpy*.rst
 docs/modules.rst
 docs/references/generated
 coverage.xml
-.coverage
+.coverage*
```
[the remark this answers is not in the document]
Comment (joshuaspear) [reply .gitignore:17]: When running tests, I seem to get .coverage... files with references to my local system. Maybe it's the way I'm running the test?

File: d3rlpy/algos/qlearning/torch/cql_impl.py (hunk 1 above)
Comment (takuseno) [reply d3rlpy/algos/qlearning/torch/cql_impl.py:81]: The interface has been updated in this commit: https://github.com/takuseno/d3rlpy/commit/67723bef91a38bb111aa6480b4d2f5ed7008eb9f . Please check.

Comment (joshuaspear) [reply d3rlpy/algos/qlearning/torch/cql_impl.py:81]: @takuseno - I'm sorry for the delay! All looks good to me except my one comment on the AWAC d3rlpy/algos/qlearning/awac.py file. Cheers

Comment (joshuaspear) [reply d3rlpy/algos/qlearning/torch/cql_impl.py:81]: @takuseno is there anything else you need from me at all for the PR? :)

Comment (takuseno) [reply d3rlpy/algos/qlearning/torch/cql_impl.py:81]: @joshuaspear Thanks for the contribution. I've changed the target branch to `refactor_loss` because I've made some new changes to master branch. I'm seeing conflicts between your PR and `refactor_loss` branch. Could you resolve it? Here is an example instruction to merge `refactor_loss` branch to your branch.
```
$ git fetch upstream
$ git checkout master
$ git merge upstream/refactor_loss
```

Comment (joshuaspear) [reply d3rlpy/algos/qlearning/torch/cql_impl.py:81]: @takuseno no probs - will do

Comment (joshuaspear) [reply d3rlpy/algos/qlearning/torch/cql_impl.py]: @takuseno have merged the branches - I included a couple more data classes for the loss outputs FYI

**Comment by takuseno** [comment]
@joshuaspear Thank you for continuing this, but, I'm seeing weirdly large number of changes in your diff now.... It could be easier to close this PR and make a new one based on the latest master to resolve this....

**Comment by joshuaspear** [comment]
@takuseno makes sense :) Will have a crack at it next week
